### ISSUE TITLE
Google Assistant Triggers with a Mic but turns off without registering the command 

### ISSUE THREAD

**Issue by farmaan999** [htruong/crankshaft#56]
Just installed the mic on my Crankshaft 0.2.0 Set up. 
But Google Assistant doesn't seem to work. 

For Example: 

Me: Hey Google
Assistant: " Assistant gets triggered and starts listening" 
Me: Navigate to Auckland Airport 
Assistant: Doesn't register anything at all, times out, and goes back to home screen. 

Specs:
Nexus 6P Android 8.1.0
Crankshaft 0.2.0
AndroidAuto 3.0.570554
Ugreen Sound card USB to 3.5mm Output

**Comment by htruong** [comment]
Yep -- You'll need to plug in a USB microphone or a DAC for it to work. [The issue is explained in details here](https://www.reddit.com/r/crankshaft/comments/85w9xo/voice_controls_usb_microphone_needed/). Unfortunately, it's Google's decision to not allow the head unit to not have a microphone, so that's something Crankshaft can't fix. :(
